index.d.ts

```diff
@@ -11,12 +13,12 @@ export class Client {
   public error(obj: LogArgument, extra?: Extra, callback?: Callback): LogResult;
   public critical(obj: LogArgument, extra?: Extra, callback?: Callback): LogResult;
 
-  public setPerson(id: string, name?: string, email?: string): void;
+  public setPerson(id: string | null, name?: string | null, email?: string | null): void;
```

Comment:
__*Note*__
Worth calling out here on name and email you actually have an effective type of `string | null | undefined` because they are optional parameters. The id field will only be `string | null` this isn't likely to cause a major problem but could affect the way you do value comparison later